Automattic/jetpack
# Blaze: only display UI to English customers.

## Proposed changes:

* We're currently only displaying the interface to English customers. We added that gating to the editor panel; let's add it to the post list actions as well.

### Other information:


## Jetpack product discussion

* 1386-gh-dotcom-forge

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

* Test on a WoA site
* Go to the Post list, in wp-admin: `wp-admin/edit.php?post_type=post`
* You should see the "Blaze" option when moving your mouse over a post row (if the post is published).
* Go to `https://wordpress.com/me/account`
* Switch your language to French and save changes
* Back on the WoA site, install the Transients Manager plugin
* Delete the `jetpack_connected_user_data_` transient
* Reload the post list in wp-admin.
* The Blaze option should not be there anymore.
